Remotion/Web/Development.WebTesting.ExecutionEngine/CompletionDetectionStrategies/Wxe.cs

```diff
@@ -25,23 +26,23 @@ public static class Wxe
   {
     public static readonly WxePostBackCompletionDetectionStrategy PostBackCompleted = new WxePostBackCompletionDetectionStrategy (1);
 
-    public static readonly Func<PageObject, WxePostBackInCompletionDetectionStrategy> PostBackCompletedIn =
-        po => PostBackCompletedInContext (po.Context);
-
     public static readonly Func<PageObjectContext, WxePostBackInCompletionDetectionStrategy> PostBackCompletedInContext =
         ctx => new WxePostBackInCompletionDetectionStrategy (ctx, 1);
 
+    public static readonly Func<PageObject, WxePostBackInCompletionDetectionStrategy> PostBackCompletedIn =
```

Comment:
Why move this down here?

Reply:
The nullability analysis requires this order to pass.